### DISCUSSION TITLE
Kibana - data and Displays missing following update

### DISCUSSION THREAD

**Discussion by trb143** [Security-Onion-Solutions/securityonion#1706] [Unsupported Versions] [answered]
Having updated to 2.3.2 and run sudo so-elastic-clear  I now have a running Kibana but do not seem to have any indexes or screens.  How can I reloaded these?

**Comment by trb143** [comment, marked as answer]
Resolved this by running curl -X POST "localhost:5601/api/saved_objects/_import?overwrite=true" -H "kbn-xsrf: true" --form file=@/opt/so/conf/kibana/saved_objects.ndjson

Found this is so-kibana-config-load.  Should this be in the documentation?

**Reply by dougburks** [reply]
Done:
https://github.com/Security-Onion-Solutions/securityonion-docs/commit/cbdcedc7cb50f31ee2c453511650caffd86b64be#diff-002308332f16e0204b75fa33df30808f5934966192a3f3b3b03f71b3af60d099
